Sources/XCTest/Private/PerformanceMeter.swift

```diff
@@ -11,7 +11,7 @@
 //  Measures the performance of a block of code and reports the results.
 //
 
-#if os(Linux) || os(FreeBSD)
+#if USE_IMPORT_FOUNDATION
```

Comment:
Instead of adding this conditional compilation flag, how about we simplify the expression to something like:

```swift
#if os(macOS)
import SwiftFoundation
#else
import Foundation
#endif
```

See also previous discussion of this: https://github.com/apple/swift-corelibs-xctest/pull/96